.github/twitter.py

```diff
@@ -0,0 +1,53 @@
+import tweepy
+import requests
+import xml.etree.ElementTree as ET
+import os
+
+
+def main():
+    client = tweepy.Client(
+        consumer_key=os.environ("API_KEY"),
+        consumer_secret=os.environ("SECRET_KEY"),
+        access_token=os.environ("ACCESS_TOKEN"),
+        access_token_secret=os.environ("ACCESS_SECRET"),
+    )
+
+    previous_atom = requests.get(
```

Comment:
eventually might be nicer to get prev atom from local git diff to remove dependency on internet but bc we need internet anyway to tweet it's pretty good